File: rtchoiceoption.pas
unit RTChoiceOption;

{$mode objfpc}

interface

uses
  Classes, SysUtils;

type
  TRTChoiceOption = class
  public
    Text: String;
    Visible: Boolean;
    VisibleIndex: Integer;

    constructor Create(AText: String);
    destructor Destroy; override;
  end;

implementation

constructor TRTChoiceOption.Create(AText: String);
begin
  inherited Create;
  Text := AText;
  Visible := true;
  VisibleIndex := 0;
end;

destructor TRTChoiceOption.Destroy;
begin
  inherited Destroy;
end;

end.

File: LORD2/RTReader/RTChoiceOption.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LORD2
{
    public class RTChoiceOption
    {
        public string Text;
        public bool Visible;
        public int VisibleIndex;

        public RTChoiceOption(string text)
        {
            Text = text;
            Visible = true;
            VisibleIndex = 0;
        }
    }
}
